prime-router/src/main/kotlin/fhirengine/translation/TranslationSchemaManager.kt

```diff
@@ -0,0 +1,154 @@
+package gov.cdc.prime.router.fhirengine.translation
+
+import ca.uhn.hl7v2.DefaultHapiContext
+import gov.cdc.prime.router.azure.BlobAccess
+import gov.cdc.prime.router.fhirengine.translation.hl7.FhirToHl7Converter
+import gov.cdc.prime.router.fhirengine.translation.hl7.FhirTransformer
+import gov.cdc.prime.router.fhirengine.utils.FhirTranscoder
+import org.apache.logging.log4j.kotlin.Logging
+
+/**
+ * Instance of this manager can be used to validate and update translation schemas in various environments
+ */
+class TranslationSchemaManager : Logging {
```

Comment:
I think it would be good to add an integration test for this feature (similar to what we discussed with HL7Passthrough) where we use a test azure blob store to prove out this functionality?

Reply:
So the tests in `TranslationSchemaManagerTests` spin up real azurite containers to test this out

Reply:
Yep this is exactly what I had in mind, I just missed it during review because it was collapsed